next/frontend/api/data.ts

```diff
@@ -0,0 +1,59 @@
+// TODO refactor, get rid of Api error, make TaxApiError format the default and handle errors accordingly
```

Comment:
would say this file serves the same function as other 'dto' files, would move it there (but ok to do that in a followup PR as well)